src/components/Header/NavBar.js

```diff
@@ -2,36 +2,45 @@ import Link from 'next/link';
 import Head from 'next/head';
 import Image from 'next/image';
 import styles from './NavBar.module.css';
+import ReusableNavLink from '../common/ReusableNavLink';
+
 
 export default function NavBar() {
+    const navList = [
+        { text: "About", href: "/about" },
+        { text: "Contact", href: "/contact" },
+      ];
+
     return (
-        <div className={styles.container}>
-            <Image
-                src="/vercel.svg"
-                alt="Vercel Logo"
-                className={styles.vercelLogo}
-                width={100}
-                height={24}
-                priority
-              />
-            
-            <form>
-                <input id="searchInputBar" type="text" placeholder="Search" name='search' required/>
-                <button type='submit'>
-                    <Image
-                        src="/search-glass.svg"
-                        alt="Search Image"
-                        className={styles.searchImage}
-                        width={20}
-                        height={20}
-                    />
-                </button>
-            </form>
-            
-            <ul className={styles.nav}>
-                <li>Create Listing</li>
-                <li>Log Out</li>
+        <div className=" relative flex items-center justify-evenly place-content-center sm:text-left md:flex py-4 px-24 lg:flex-row flex-col">
+            <Link href="/">
+                <Image
+                    src="/LOGO.png"
+                    alt="Find Your People Tech Logo"
+                    className="sm:h-full"
+                    width={80}
+                    height={28}
+                    priority
+                />
+            </Link>
+            <ul className="flex items-center justify-between gap-10">
```

Comment:
@istoga from line 26 to 43 needs to be wrapped in a div/container so the justify content in line 15 can take effect 
also this container has to properly position and align(add a flex class)

Reply:
Done

Reply:
@istoga Good job
I didn't mean container as an element, sorry for the mix up
Kindly change it to a div element

Reply:
Removed container and made into div element.